Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿Console.WriteLine("Введите число m");
+int m = int.Parse(Console.ReadLine());
+Console.WriteLine("Введите число n");
+int n = int.Parse(Console.ReadLine());
+Random rand = new Random();
+double[,] array = new double[m,n];
+if(m>=5 & m<=20 & n>=5 & n<=20)
+{
+    for(int i =0; i<m; i++)
+    {
+        for(int j =0; j<n; j++)
+        {
+            array[i,j] = rand.Next(0,100);
+        }
+    }
+    static void vuvod(double[,] array, int m, int n)
+    {
+        for(int i = 0; i < m; i++)
+        {
+            for(int j =0; j<n;j++)
+            {
+                Console.Write(array[i,j]+" ");
+            }
+            Console.WriteLine(" ");
+        }
+    }
+    vuvod(array,m,n);
+    Console.WriteLine("Введите число");
+    double chislo = double.Parse(Console.ReadLine());
+    static double[,] poisk(double[,] array, int m, int n, double l)

```

Comment:
Стиль! .Именуйте методы с большой буквы.
```suggestion
    static double[,] Poisk(double[,] array, int m, int n, double l)

```